src/code/z_camera.c

```diff
@@ -2428,58 +2560,62 @@ s32 Camera_Normal3(Camera* camera) {
         sp80.yaw = sp68.yaw + sp90;
     }
 
-    if (dynamicData->yawTimer > 0) {
-        sp80.yaw += dynamicData->yawUpdateRate;
-        dynamicData->yawTimer--;
-        if (dynamicData->yawTimer == 0) {
-            Camera_UnsetFlags(camera, CAM_FLAG2_20);
+    if (rwData->yawTimer > 0) {
+        sp80.yaw += rwData->yawUpdateRate;
+        rwData->yawTimer--;
+        if (rwData->yawTimer == 0) {
+            Camera_UnsetStateFlag(camera, CAM_STATE_DISABLE_MODE_CHANGE);
         }
     }
 
     OLib_VecSphAddToVec3f(eyeNext, at, &sp80);
+
     if (camera->status == CAM_STATUS_ACTIVE) {
         *eye = *eyeNext;
         func_800CBFA4(camera, at, eye, 0);
     } else {
         *eye = *eyeNext;
     }
 
-    camera->fov = Camera_LERPCeilF(fixedData->fovTarget, camera->fov, camera->fovUpdateRate, 0.1f);
-    if (fixedData->flags & NORM3_FLG_20) {
-        camera->roll = Camera_LERPCeilS(0, camera->roll, 0.05f, 5);
+    camera->fov = Camera_ScaledStepToCeilF(roData->fovTarget, camera->fov, camera->fovUpdateRate, 0.1f);
+
+    if (roData->interfaceFlags & NORMAL3_FLAG_5) {
+        camera->roll = Camera_ScaledStepToCeilS(0, camera->roll, 0.05f, 5);
     } else {
-        camera->roll = Camera_LERPCeilS(0, camera->roll, 0.1f, 5);
+        camera->roll = Camera_ScaledStepToCeilS(0, camera->roll, 0.1f, 5);
     }
 
-    camera->atLERPStepScale = Camera_ClampLERPScale(camera, fixedData->maxAtLERPScale);
-    dynamicData->flag &= ~1;
-    return true;
+    camera->atLerpStepScale = Camera_ClampLerpScale(camera, roData->maxAtLERPScale);
+    rwData->flag &= ~1;
```

Comment:
This is a flag entirely internal to this function (and separate to the `NORMAL3_FLAG_`'s), what should I call it? `NORMAL3_RW_FLAG`?

Reply:
Oh, only here? damn, that's tricky.
If that bit is never set anywhere then giving it a name is kinda pointless. I guess we should just leave it raw and leave a comment explaining it a bit

Reply:
The bit is set earlier in the function, it's just all internal. The only reason `NORMAL3_FLAG_` isn't internal is because the data that sets the flag is external in `camera_data.c`, while this bit is set internally.

I did end up adding a flag define for it